packages/frontend/src/components/dialogs/CreateChat/index.tsx

```diff
@@ -199,7 +199,16 @@ function CreateChatMain(props: CreateChatMainProps) {
   // Chatmail accounts can't send unencrypted emails. See
   // - https://github.com/deltachat/deltachat-desktop/issues/5294#issuecomment-3089552788
   // - https://github.com/deltachat/deltachat-ios/blob/a0043be425d9c14f4039561957adb82ef1ab2adb/deltachat-ios/Controller/NewChatViewController.swift#L76-L78
-  const showNewEmail = !isChatmail && queryStr.length === 0
+  const [forceShowNewEmail_, setForceShowNewEmail_] = useState(false)
```

Comment:
I think we should not add functions in the production code just for testing. Only data-test attributes if possible.

We have no secrets for creating real non chatmail accounts, so maybe better use these

Reply:
It's just a simple toggle to hide or show a button, which is clearly defined.
I mean, in a perfect world we wouldn't need it, but don't think it's that bad.
Otherwise if the server goes down, or if you don't know the secret (e.g. run tests locally), the test will not work.

I suggest to keep this test and add another one, maybe reuse the code.